docs/OPTUM_PANTHER/Optum_Panther_Microbiology_STEM.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Microbiology
+nav_order: 4
+parent: Optum EHR to STEM
+grand_parent: Optum EHR
+description: "OPTUM EHR Microbiology table to STEM"
+---
+
+# CDM Table name: STEM
+
+## Reading from OPTUM_EHR.Microbiology
+
+|     Destination Field    |     Source Field    |     Logic    |     Comment    |
+|-|-|-|-|
+| id | autogenerate  | | |
+| domain_id | Observation | | |
+| person_id | ptid | | |
+| visit_occurrence_id | encid | Lookup the VISIT_OCCURRENCE_ID based on the encid |If encid is blank then use coalesce(collect_date, result_date) to determine which VISIT_OCCURRENCE_ID the record should be associated to|
+| visit_detail_id| encid | Lookup the VISIT_DETAIL_ID based on the encid|If encid is blank then leave VISIT_DETAIL_ID blank|
+| provider_id |  encid | Lookup the PROVIDER_ID from the VISIT_DETAIL table using the encid|If encid is blank then leave PROVIDER_ID blank|
+| start_date | coalesce(collect_date, result_date)  | | |
+| end_date | coalesce(collect_date, result_date) | | | 
+| start_datetime | coalesce(collect_date, result_date) collect_time | Combine the coalesce(collect_date, result_date) and collect_time to create a datetime| |
+| end_datetime | coalesce(collect_date, result_date) collect_time | Combine the coalesce(collect_date, result_date) and collect_time to create a datetime| |
+| concept_id | 4155370 | Microbiology | |
```

Comment:
Let's change this to 4252364 - Clinical Microbiology

Reply:
concept_id was updated